### PR TITLE
Internal: Added missing E2E tests for video widget [ED-11033]

### PR DESCRIPTION
**Pull request by dmaziar**
## PR Checklist

- [ ] The commit message follows our guidelines:  https://github.com/elementor/elementor/blob/master/.github/CONTRIBUTING.md

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ x] Other... Please describe: playwright tests

## Summary

This PR can be summarized in the following changelog entry:

*

## Description
An explanation of what is done in this PR

*

## Test instructions
This PR can be tested by following these steps:

*

## Quality assurance

- [ ] I have tested this code to the best of my abilities
- [ ] I have added unittests to verify the code works as intended
- [ ] Docs have been added / updated (for bug fixes / features)

Fixes #

### CODE REVIEW

File: tests/playwright/pages/elementor-panel-tabs/content.js [hunk 1]
```diff
@@ -12,18 +12,23 @@ export default class Content {
...
 		if ( options.linkTo ) {
 			await this.selectLinkSource( 'Custom URL' );
 		}
 
-		const urlInput = this.page.locator( EditorSelectors.button.url );
+		const urlInput = this.page.locator( options.linkInpSelector );
 		await urlInput.clear();
 		await urlInput.type( link );
-		await this.page.locator( EditorSelectors.button.linkOptions ).click();
+		const wheel = await this.page.locator( EditorSelectors.button.linkOptions );
```
Comment (yotamselementor) [inline tests/playwright/pages/elementor-panel-tabs/content.js:24]:
```suggestion
		const wheel = this.page.locator( EditorSelectors.button.linkOptions );
```

File: tests/playwright/sanity/video.test.js [hunk 2]
```diff
@@ -44,6 +48,85 @@ test.describe( 'Video tests inside a container', () => {
...
+			if ( 'youtube' === video ) {
+				await videoWidget.setTime( 'End', endTime );
+				await videoWidget.selectSuggestedVideos( 'Any Video' );
+			}
+
+			const controls = await new Promise( ( resolved ) => {
+				resolved( player.controls.map( ( control ) => {
+					return EditorSelectors.video[ control ];
+				} ) );
+			} );
```
Comment (yotamselementor) [inline tests/playwright/sanity/video.test.js:78]: I don't think we need a `Promise` here:
```suggestion
			const controls = player.controls.map( ( control ) => {
				return EditorSelectors.video[ control ];
			} );
```

**Review by yotamselementor — CHANGES_REQUESTED** [review]

File: tests/playwright/pages/elementor-panel-tabs/content.js (hunk 1 above)
Comment (dmaziar) [reply tests/playwright/pages/elementor-panel-tabs/content.js]: fixed

File: tests/playwright/sanity/video.test.js (hunk 2 above)
Comment (dmaziar) [reply tests/playwright/sanity/video.test.js]: fixed

**Review by yotamselementor — APPROVED** [review]

**Pull request merged**